docs/Spark/fabrics/dataproc/dataproc-tips.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Troubleshooting Tips"
```

Comment:
should we specify what is being troubleshoot-ed in the title? or do you think this page will be an all-in-one troubleshooting page if there are more issues?

Reply:
How's this title: 
"Connectivity Tips"